bin/phpunit-each

```diff
@@ -11,7 +11,16 @@ $hasError = 0;
 
 chdir("$srcDir/tools");
 foreach ($args as $test) {
-  $cmd = sprintf("./scripts/phpunit --tap --log-junit %s %s %s",
+  if (file_exists("../Civi/Test/TAP.php")) {
+    $printCMD = "--printer '\Civi\Test\TAP'";
+  }
+  elseif (class_exists('PHPUnit_Util_Log_TAP')) {
+    $printCMD = "--tap";
+  }
+  else {
+    $printCMD = "--debug";
+  }
```

Comment:
I think there's a basic question "*How important is TAP format*?" There are a few different reasons one might use `--tap`:

1. It's handy for a person skimming the output (esp when trying to gauge progress and/or to investigate PHP fatals).
2. It's a formal spec which can be integrated/scripted with other tools. In this case, it's part of a technical contract.

If `#1` completely captures our usage, then we have a lot of flexibility. It's acceptable to have some mid-tier scripts which switch around between output formats (e.g. `--tap` vs `--debug`). It may even be preferable to change from TAP format to a custom format (borrowing good bits from `--tap` and `--debug`). OTOH, if it's `#2`, then other scripts would expect the output to be strictly TAP -- so switching to anything else (even if it's a fallback like `else {$printCMD = '--debug';}`) would break the contract.